src/macaron/slsa_analyzer/package_registry/pypi_registry.py

```diff
@@ -305,3 +271,146 @@ def get_maintainer_join_date(self, username: str) -> datetime | None:
         res: datetime | None = parse_datetime(datetime_val, datetime_format)
 
         return res.replace(tzinfo=None) if res else None
+
+
+@dataclass
+class PyPIPackageJsonAsset:
+    """The package JSON hosted on the PyPI registry.
+
+    This class matches the AssetLocator protocol.
+    """
+
+    #: The target pypi software component.
+    component: Component
+
+    #: The pypi registry.
+    pypi_registry: PyPIRegistry
+
+    #: The size of the asset (in bytes). This attribute is added to match the AssetLocator
+    #: protocol and is not used because pypi API registry does not provide it.
+    size_in_bytes: int
+
+    #: The asset content.
+    package_json: dict
+
+    @property
+    def name(self) -> str:
+        """Get the asset name."""
+        return "package_json"
```

Comment:
I wonder if this is intended to be a constant for all instance of `PyPIPackageJsonAsset`?

Reply:
yes